### PR TITLE
Merge

### PR DESCRIPTION
**Pull request by cosa65**
# Description

# Details
#### Links to any PRs or resources related to this PR
https://github.com/hms-dbmi-cellenics/pipeline/pull/312
https://github.com/hms-dbmi-cellenics/worker/pull/307
https://github.com/hms-dbmi-cellenics/api/pull/442
https://github.com/hms-dbmi-cellenics/ui/pull/848

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [ ] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [ ] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [ ] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [ ] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [ ] Started end-to-end tests on the latest commit.

### Documentation updates
- [ ] Relevant Github READMEs updated **or** no GitHub README updates required.
- [ ] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

File: README.md
Comment (alexvpickering) [inline README.md:1]: why are these references for `biomage-org`?

File: .github/workflows/pr_validate.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
       - id: check-issue-url
         name: Check issue URL
         run: |-
-          REGEX="#### URL to issue\s?\r\n(https:\/\/github\.com\/hms-dbmi-cellenics\/issues\/issues\/[0-9]+|N\/A)"
+          REGEX="#### URL to issue\s?\r\n(https:\/\/github\.com\/hms-dbmi-cellenics\/issues\/issues\/[0-9]+|https:\/\/biomage\.atlassian\.net\/browse\/BIOMAGE-[0-9]+|https:\/\/github\.com\/biomage-org\/issues\/issues\/[0-9]+|N\/A)"
```
Comment (alexvpickering) [inline .github/workflows/pr_validate.yaml]: Why is it necessary here to check for issues in either `biomage-org` or Biomage's private Jira (which is not publicly viewable)?

File: python/README.md
Comment (alexvpickering) [inline python/README.md:1]: why are there references to `biomage-org` here?

File: README.md
Comment (cosa65) [reply README.md:1]: I am correcting these right now, changing them over to hms-dbmi-cellenics

**Review by alexvpickering — APPROVED** [review]
looks good

**Pull request merged**